playwright/tests/test_17_EstTool_UI_AssumpLimitations.py

```diff
@@ -96,7 +96,8 @@ def test_example(page: Page) -> None:
     expect(page.get_by_text("These figures are combined")).to_be_visible()
     expect(page.get_by_role("heading", name="Network Data Transfer")).to_be_visible()
     expect(page.get_by_text("Our outgoing network data")).to_be_visible()
-    expect(page.get_by_text("Assumptions and Limitations The Technology Carbon Estimator tool is designed to")).to_be_visible()
+    expect(page.get_by_text("Assumptions and Limitations")).to_be_visible()
```

Comment:
I needed to break this up because it was failing. I think it was because there is now an 'X' button between the first and second sentences.